src/main/java/ch/njol/skript/effects/EffCancelEvent.java

```diff
@@ -42,39 +13,56 @@
 import ch.njol.skript.log.ErrorQuality;
 import ch.njol.skript.util.Utils;
 import ch.njol.util.Kleenean;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.block.BlockCanBuildEvent;
+import org.bukkit.event.entity.EntityToggleSwimEvent;
+import org.bukkit.event.inventory.InventoryInteractEvent;
+import org.bukkit.event.player.PlayerDropItemEvent;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.event.player.PlayerLoginEvent;
+import org.jetbrains.annotations.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Cancel Event")
 @Description("Cancels the event (e.g. prevent blocks from being placed, or damage being taken).")
-@Examples({"on damage:",
-		"	victim is a player",
-		"	victim has the permission \"skript.god\"",
-		"	cancel the event"})
+@Examples({
+	"on damage:",
+		"\tvictim is a player",
+		"\tvictim has the permission \"skript.god\"",
+		"\tcancel the event"
+})
 @Since("1.0")
 public class EffCancelEvent extends Effect {
+
 	static {
 		Skript.registerEffect(EffCancelEvent.class, "cancel [the] event", "uncancel [the] event");
 	}
 	
 	private boolean cancel;
-	
-	@SuppressWarnings("null")
+
 	@Override
-	public boolean init(final Expression<?>[] vars, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+	public boolean init(Expression<?>[] expressions, int matchedPattern,
+						Kleenean isDelayed, ParseResult parseResult) {
 		if (isDelayed == Kleenean.TRUE) {
-			Skript.error("Can't cancel an event anymore after it has already passed", ErrorQuality.SEMANTIC_ERROR);
+			Skript.error("Can't cancel an event after it has already passed", ErrorQuality.SEMANTIC_ERROR);
```

Comment:
```suggestion
			Skript.error("An event cannot be cancelled after it has already passed", ErrorQuality.SEMANTIC_ERROR);
```
I think this is a better statement to make